docs/recipes/echam_disturbance.rst

```diff
@@ -0,0 +1,73 @@
+Applying a temporary disturbance to ECHAM
```

Comment:
Through out the document we follow the convention of capitalizing the everything that is not an article or a preposition. I am not the biggest fan of it but should we stick to it?

Reply:
Christian now suggested an even longer title...Maybe we should find a nice system for this?